src/Microsoft.Identity.Client/ClientApplicationBase.cs

```diff
@@ -160,45 +147,29 @@ internal ITelemetryReceiver TelemetryReceiver
         /// Unless requested otherwise by Microsoft support, this parameter should not be set by application developers as it may have adverse effect on the application.
         /// This property is also concatenated to the <c>extraQueryParameter</c> parameters of token acquisition operations.
         /// </summary>
-        public string SliceParameters { get; set; }
+        public string SliceParameters => ServiceBundle.Config.SliceParameters;
```

Comment:
I think it's a good candidate to remove from here and expose only on the Config object.

Reply:
Given other discussion of not exposing IApplicationConfiguraiton as a public interface, keeping some of the properties here readonly makes some sense.   I'd love feedback from JeanMarc/Henrik on whether this needs to be on clientapplicationbase at all.  I'm not sure what value it provides to developers.

---
In reply to: [241516638](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/723#discussion_r241516638) [](ancestors = 241516638)